tests/network/flat_overlay/conftest.py

```
@@ -51,6 +52,19 @@
 SPECIFIC_HOST_MASK = "32"
 
 
+def restart_ovnkube_node_daemonset():
+    subprocess.check_call(["oc", "rollout", "restart", "daemonset/ovnkube-node", "-n", "openshift-ovn-kubernetes"])
```

Ruff (0.15.18)

[error] Starting a process with a partial executable path

(S607)